### PR TITLE
[Account] Subscribe to the newsletter in customer account

### PR DESCRIPTION
**Pull request by GSadee**
| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    | yes
| BC breaks?      | no
| Related tickets | 
| License         | MIT

### CODE REVIEW

File: src/Sylius/Behat/Page/Shop/Account/ProfileUpdatePage.php [hunk 1]
```diff
@@ -76,6 +76,22 @@ public function saveChanges()
     /**
      * {@inheritdoc}
      */
```
Comment (Zales0123) [inline src/Sylius/Behat/Page/Shop/Account/ProfileUpdatePage.php]: Why do we need a ``{@inheritdoc}`` if there is no docblock in interface?

File: src/Sylius/Component/Customer/Model/Customer.php [hunk 2]
```diff
@@ -266,6 +271,22 @@ public function setPhoneNumber($phoneNumber)
         $this->phoneNumber = $phoneNumber;
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function isNewsletterSubscription()
```
Comment (pjedrzejewski) [inline src/Sylius/Component/Customer/Model/Customer.php]: isSubscribedToNewsletter? (we can skip "The")
protected $subscribedToNewsletter = false;

**Comment by pjedrzejewski** [comment]
@GSadee Nice, please tweak the field name and it is good to go!

**Comment by pjedrzejewski** [comment]
Thank you Grzesiu! :)

**Pull request merged**